tcs/base/fn_adminInit.sqf

```diff
@@ -12,13 +12,15 @@
 
 private _start = time;
 private _adminIds = [
-	"76561198038553572", // Hingle
+	"76561198037776313", // BearMilitia
+	"76561197982851011", // Dumbname
+	"76561198074598707", // FableChaser
+	"76561193743586276", // Hingle
 	"76561198028549221", // Lucas
-	"76561197992043612", // Science
-	"76561197981421300", // Metkill
 	"76561197997429145", // M16
-	"76561197982851011", // Dumbname
-	"76561198042629560" // Pliskin
+	"76561198064974942", // MexterInfinite
+	"76561197992043612", // Science
+	"76561198052584606"  // Zapper
 ];

```

Comment:
Could we replace this with: 

```sqf
private _adminIds = getArray (missionConfig >> "enableDebugConsole");
```

So we don't have to remember to change it in several places next time?

Reply:
Updated; there were no noticeable errors on the server.